coalib/results/TextPosition.py

```diff
@@ -33,6 +33,19 @@ def column(self):
         return self._column
 
     @classmethod
+    def calc_position(self, text, line, column):
+        """
+        Calculates position in a string from line number and column.
+        Uses \\n  as the newline character. Position starts from 0
+        """
+        lines = text.splitlines(keepends=True)
```

Comment:
:( We *could* make this algo better and faster me thinks.

BTW: Just so you know, 58570fbc5cfcf418d5a8c8134942042c2476b7fd also has a simpler algo which is doing `.splitlines(True)` and then using cumulative sum until you get the line, and then using the left over position to get the column

We probably should `timeit` them both and check efficiency :) (But this can go into another issue)